src/features/rewards/profile/style.ts

```diff
@@ -57,13 +57,12 @@ export const StyledContent = styled<Partial<Props>, 'div'>('div')`
 `
 
 export const StyledTitleWrap = styled<Partial<Props>, 'div'>('div')`
+  width: 280px;
```

Comment:
I always prefer to use max-width then width. This allows responsive design in case we want to resize panel